poly-synth/src/poly-synth-demo.js

```diff
@@ -61,17 +70,32 @@ class PolySynthDemo {
     this.initializeAmplifierEnvelopeGUI_(identifiers.gainEnvelopeDivId);
     this.initializeFilterGUI_(identifiers.filterDivId);
     this.initializeFilterEnvelopeGUI_(identifiers.filterEnvelopeDivId);
+    this.initializeSideChainGUI_(identifiers.sideChainDivId);
     this.initializeBitcrusherGUI_(identifiers.bitcrusherDivId);
     this.initializeDelayGUI_(identifiers.delayDivId);
     this.initializeReverbGUI_(identifiers.reverbDivId, this.reverbSelectorId_);
     this.initializeMasterVolumeGUI_(identifiers.volumeDivId);
-    
+
     this.loadImpulseResponses(this.impulseResponseUrls_)
         .then((impulseResponseBuffers) => {
           this.impulseResponseBuffers_ = impulseResponseBuffers;
           this.polySynth_.setConvolverBuffer(this.impulseResponseBuffers_[0]);
           document.getElementById(this.reverbSelectorId_).disabled = false;
         });
+
+    this.loadDrumSamples(this.drumSampleUrls_)
+        .then((drumSampleBuffers) => {
+          this.drumSampleBuffers_ = drumSampleBuffers;
+          document.getElementById(this.drumSelectorId_).disabled = false;
+        });
+  }
+
+  async loadDrumSamples(drumSamples){
+    let drumSampleBuffers = [];
+    for (let index in drumSamples) {
+      drumSampleBuffers[index] = await this.loadSound(drumSamples[index]);
+    }
+    return drumSampleBuffers;
```

Comment:
Don't you do exactly the same for the reverb?  It would be nice to factor out this common code.